ops/__init__.py

```diff
@@ -12,31 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""The ops library: a Python framework for writing Juju charms.
+"""The API to respond to Juju events and manage the application.
```

Comment:
What's the reasoning behind this change? Maybe it's just what I'm used to, but the previous description seems more accurate to me.

Reply:
I initially agreed with Tony, but in the rendered docs that now has an intro, I can see why this is better on the ops module now.

Reply:
Oh, I missed that the original text had moved - I thought that it was removed. I guess it's a reasonable way to describe the names exposed in the `ops` namespace (as opposed to describing the ops package itself).